missing schema.json for 4.48.1 release

as #1187 is closed and the issue still persists, raise this new issue instead.

error log as below:

```
==> /opt/homebrew/Cellar/gitlab-ci-local/4.48.1/bin/gitlab-ci-local --list
Error: Cannot find module './schema.json'
Require stack:
- /opt/homebrew/Cellar/gitlab-ci-local/4.48.1/libexec/lib/node_modules/gitlab-ci-local/src/schema/index.js
- /opt/homebrew/Cellar/gitlab-ci-local/4.48.1/libexec/lib/node_modules/gitlab-ci-local/src/validator.js
- /opt/homebrew/Cellar/gitlab-ci-local/4.48.1/libexec/lib/node_modules/gitlab-ci-local/src/parser.js
- /opt/homebrew/Cellar/gitlab-ci-local/4.48.1/libexec/lib/node_modules/gitlab-ci-local/src/index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1028:15)
    at Function.Module._load (node:internal/modules/cjs/loader:873:27)
    at Module.require (node:internal/modules/cjs/loader:1100:19)
    at require (node:internal/modules/cjs/helpers:119:18)
    at Object.<anonymous> (/opt/homebrew/Cellar/gitlab-ci-local/4.48.1/libexec/lib/node_modules/gitlab-ci-local/src/schema/index.ts:2:16)
    at Module._compile (node:internal/modules/cjs/loader:1198:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1252:10)
    at Module.load (node:internal/modules/cjs/loader:1076:32)
    at Function.Module._load (node:internal/modules/cjs/loader:911:12)
    at Module.require (node:internal/modules/cjs/loader:1100:19)

/opt/homebrew/Library/Homebrew/ignorable.rb:29:in `block in raise'
Minitest::Assertion: Expected: 0
  Actual: 1
```

cc @firecow

relates to https://github.com/Homebrew/homebrew-core/pull/169393

--- a/.npmignore
+++ b/.npmignore
@@ -1,4 +1,5 @@
 *
+!src/schema/schema.json
 !src/**/*.js
 !src/**/*.d.ts
 src/tests/**/*.js

